DEVELOPING.md

```diff
@@ -172,6 +172,6 @@ following incantations.
 
     > ;ir2_12/publishLocal;linkerInterface2_12/publishLocal;linker2_12/publishLocal;testAdapter2_12/publishLocal;sbtPlugin/publishLocal
     > ++SCALA_VERSION
-    > ;compiler2_12/publishLocal;library2_12/publishLocal;testInterface2_12/publishLocal;testBridge2_12/publishLocal;jUnitRuntime2_12/publishLocal;jUnitPlugin2_12/publishLocal
+    > ;compiler2_12/publishLocal;library2_12/publishLocal;javalib/publishLocal;javalibintf/publishLocal;testInterface2_12/publishLocal;testBridge2_12/publishLocal;jUnitRuntime2_12/publishLocal;jUnitPlugin2_12/publishLocal
```

Comment:
Wouldn't it make more sense to put it in the first list, which is independent of the version of Scala?

Reply:
Ah, good point. In fact, it seems the second line isn't entirely correct: IIUC only the compiler should have the fully pinned version (for a build as close to a release build as possible). The others should have the default minor Scala version.

Reply:
I added a second commit with this.